code/modules/food_and_drinks/food/foods/junkfood.dm

```diff
@@ -121,6 +121,16 @@
 	user.put_in_hands(L)
 	qdel(src)
 
+/obj/item/food/chocolate_bar
+	name = "Delux Chocolate-bar"
+	desc = "Crafted from premium cacao and infused with quantum nutrients, this luxury chocolate bar is wrapped in nanofoil for ultimate freshness. Experience the essence of culinary supremacy with every bite."
+	icon_state = "choco"
```

Comment:
```suggestion
	icon = 'icons/obj/food/candy.dmi'
	icon_state = "choco"
```
Currently the sprite is invisible, gotta point to the correct dmi file.

Reply:
It should be working now , but it was working just fine in my previous test.